Peer/App/AppBuilder/CommandLineParser.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using CommandLine;
+using Peer.Parsing.CommandLine;
+
+namespace Peer.App.AppBuilder;
+
+public class CommandLineParser
+{
+    private readonly List<IVerb> _verbs;
+
+    public CommandLineParser(IEnumerable<IVerb> verbs)
+    {
+        _verbs = verbs.ToList();
+    }
+
+    public ParserResult<object> Parse(string[] args)
```

Comment:
This is probably the wrongest bit of code in here - I was enjoying wine.

Reply:
aha yea the flow through here is interesting